product_price_history/models/product_template.py

```diff
@@ -4,6 +4,8 @@
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
+    cost = fields.Float(group_operator="avg")
```

Comment:
Is it a good idea to have the value of cost propagate to the product variant if there is only one variant ?

Reply:
Indeed. I have to take a look.

Reply:
Hi @gbrito-erpopen. I updated the code. I made an error, I wanted to set the group_operator on the ``cost`` field of the ``product.price.history`` model.
Now fixed.

Reply:
@legalsylvain looks good, makes more sense :)